com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -638,7 +642,7 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
 
         if (xrActive)
         {
-            CommandBuffer cmd = CommandBufferPool.Get();
+            CommandBuffer cmd = CommandBufferPool.Get(); // pipelineCommandBuffer; //CommandBufferPool.Get();
```

Comment:
Why do we still call get here in this path instead of getting the pipelineCommandBuffer?